boards/arm/ebyte_e73_tbb_nrf52832/ebyte_e73_tbb_nrf52832-pinctrl.dtsi

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2022 morsisko
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&pinctrl {
+	uart0_default: uart0_default {
+		group1 {
+			psels = <NRF_PSEL(UART_TX, 0, 6)>,
+					<NRF_PSEL(UART_RX, 0, 8)>,
+					<NRF_PSEL(UART_RTS, 0, 5)>,
+					<NRF_PSEL(UART_CTS, 0, 7)>;
+		};
+	};
+
+	uart0_sleep: uart0_sleep {
+		group1 {
+			psels = <NRF_PSEL(UART_TX, 0, 6)>,
+					<NRF_PSEL(UART_RX, 0, 8)>,
+					<NRF_PSEL(UART_RTS, 0, 5)>,
+					<NRF_PSEL(UART_CTS, 0, 7)>;
```

Comment:
there's still an extra tab
```suggestion
			psels = <NRF_PSEL(UART_TX, 0, 6)>,
				<NRF_PSEL(UART_RX, 0, 8)>,
				<NRF_PSEL(UART_RTS, 0, 5)>,
				<NRF_PSEL(UART_CTS, 0, 7)>;
		};
	};

	uart0_sleep: uart0_sleep {
		group1 {
			psels = <NRF_PSEL(UART_TX, 0, 6)>,
				<NRF_PSEL(UART_RX, 0, 8)>,
				<NRF_PSEL(UART_RTS, 0, 5)>,
				<NRF_PSEL(UART_CTS, 0, 7)>;
```

Reply:
I think there is some strange isssue with the github tab wrapping. Before the last change that is how it looked for me:
![firefox_2022-04-18_11-01-00](https://user-images.githubusercontent.com/11231925/163784984-8f29344d-0b4b-4488-80fd-9232beb06b82.png)
Now after deleting the single tab you requested it looks like this:
![obraz](https://user-images.githubusercontent.com/11231925/163785043-1d98fb92-a07b-4b1e-a1fd-51aaeda3ea48.png)